src/compiler/datetime.rs

```diff
@@ -33,13 +34,18 @@ impl TimeZone {
     ///
     /// Returns parse errors from the underlying time parsing functions.
     pub fn datetime_from_str(&self, s: &str, format: &str) -> Result<DateTime<Utc>, ParseError> {
+        let mut parsed = Parsed::new();
+        parse(&mut parsed, s, StrftimeItems::new(format))?;
+
         match self {
-            Self::Local => Local
-                .datetime_from_str(s, format)
-                .map(|dt| datetime_to_utc(&dt)),
-            Self::Named(tz) => tz
-                .datetime_from_str(s, format)
```

Comment:
Some context here: 
The suggested replacement is [this](https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.parse_from_str). It states that timezones must be provided so it cannot be used a straight up replacement here.